airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -341,6 +343,14 @@ def is_builtin_type(cls) -> bool:
             return False
         return cls.__module__ == "builtins"
 
+    @staticmethod
+    def _extract_missing_parameters(error: TypeError) -> List[str]:
+        parameter_search = re.search(r"keyword-only.*:\s(.*)", str(error))
```

Comment:
we wild card things after `keyword-only` because depending on the number of missing parameters, it either comes as `keyword-only argument` or `keyword-only arguments` (plural)